### DISCUSSION TITLE
About install internal network

### DISCUSSION THREAD

**Discussion by nanoziobelo** [amidaware/tacticalrmm#1471] [Q&A]
Hi
I want try to install to use it on internal network class192.168.0.1/24 without external connection is possible?

**Comment by dinger1986** [comment]
Yes but you still need to use an internet resolvable domain name, you don't need to open ports

**Comment by nanoziobelo** [comment]
I see installer ask internet connection and from mai lan cant access to internet.
To dns I can make internal dns server. but cant solve to get info from internet.

**Reply by dinger1986** [reply]
It does need an outgoing connection to verify the ssl certificates using certbot and internet resolvable dns (a real domain).

**Reply by dinger1986** [reply]
Alternatively you can get the ssl certificates from elsewhere and have them in in place, comment out certbot commands etc, but it needs to have internet to download packages etc

**Comment by nanoziobelo** [comment]
Is possible to make this:
1 creata machine with internet access with lan 192..... and install al system
2 clone disk on other machine and change ip 192....  to my internal network 10...... ? or in case chenge the ip of install machine in 10....?

**Reply by dinger1986** [reply]
Yea of course, just change your dns

**Comment by nanoziobelo** [comment]
I try to remake a clean debia install machine edit the hosts file like
`
192.168.1.103   WEBRRM.localhost WEBRRM api.WEBRRM.localhost rmm.WEBRRM.localhost mesh.WEBRRM.localhost
`

get this error

The server will not issue certificates for the identifier :: Error creating new order :: Cannot issue for "*.webrrm.localhost": Domain name does not end with a valid public suffix (TLD)
think problem on generate certificate I can force to skip?
